src/mixins/itext_behavior.mixin.ts

```diff
@@ -193,7 +132,7 @@ export abstract class ITextBehaviorMixin<
       },
     };
 
-    obj.animate('_currentCursorOpacity', targetOpacity, {
+    obj._animate('_currentCursorOpacity', targetOpacity, {
```

Comment:
Just to be clear this fixes the cursor regression,
I don't mind PRing this line, just had to fix it to see cursor works with this PR changes

Reply:
#8589